src/main/java/com/kylenicholls/stash/parameterizedbuilds/ciserver/Jenkins.java

```diff
@@ -287,7 +290,18 @@ public JenkinsResponse triggerJob(String projectKey, ApplicationUser user, Job j
 			}
 		}
 
-		return sanitizeTrigger(buildUrl, joinedUserToken, prompt);
+        String csrfHeader = null;
+        if (jenkinsServer.getCsrfEnabled()) {
+            // get a CSRF token because cross site protection is enabled
+            try {
+                logger.debug("get csrf header with token: "+joinedUserToken);
```

Comment:
Even if it is only available in debug mode, I'm not a fan of writing a user's api token to the logs. I'd prefer that this logging instance was removed.

Reply:
cannot be more than agree